### PR TITLE
Read std mean from file

### PR DESCRIPTION
**Pull request by Joao-L-S-Almeida**
As the number of bands can be considerably large, it could be more convenient to input mean and std info via text files.

### CODE REVIEW

File: terratorch/datamodules/generic_scalar_label_data_module.py [hunk 1]
```diff
@@ -27,6 +27,21 @@ def wrap_in_compose_is_list(transform_list):
     # set check shapes to false because of the multitemporal case
     return A.Compose(transform_list, is_check_shapes=False) if isinstance(transform_list, Iterable) else transform_list
 
+def load_from_file_or_attribute(value:list[float] | str):
+
+    if type(value) == list:
+        return value
+    elif type(str): # It can be the path for a file
```
Comment (CarlosGomes98) [inline terratorch/datamodules/generic_scalar_label_data_module.py:34]: prefer `isinstance over `type`

File: terratorch/datamodules/generic_scalar_label_data_module.py [hunk 2]
```diff
@@ -27,6 +27,21 @@ def wrap_in_compose_is_list(transform_list):
     # set check shapes to false because of the multitemporal case
     return A.Compose(transform_list, is_check_shapes=False) if isinstance(transform_list, Iterable) else transform_list
 
+def load_from_file_or_attribute(value:list[float] | str):
```
Comment (CarlosGomes98) [inline terratorch/datamodules/generic_scalar_label_data_module.py:30]: weird spacing in argument

File: terratorch/datamodules/generic_scalar_label_data_module.py [hunk 3]
```diff
@@ -27,6 +27,21 @@ def wrap_in_compose_is_list(transform_list):
...
+    if type(value) == list:
+        return value
+    elif type(str): # It can be the path for a file
+        if os.path.isfile(value):
+            try:
+                content = np.genfromtxt(value).tolist()
+            except:
+                raise Exception(f"File must be txt, but received {value}")
+        else:
+            raise Exception("It seems that {value} does not exist or is not a file.")
```
Comment (CarlosGomes98) [inline terratorch/datamodules/generic_scalar_label_data_module.py:41]: Missing f string. Also, exception messages are usually in a more imperative tone, in this case I would remove "It seems that"

**Review by CarlosGomes98 — CHANGES_REQUESTED** [review]

**Comment by Joao-L-S-Almeida** [comment]
@CarlosGomes98 They were solved.

File: terratorch/datamodules/generic_pixel_wise_data_module.py [hunk 4]
```diff
@@ -23,6 +23,20 @@ def wrap_in_compose_is_list(transform_list):
...
+    if type(value) == list:
+        return value
+    elif type(str): # It can be the path for a file
+        if os.path.isfile(value):
+            try:
+                content = np.genfromtxt(value).tolist()
+            except:
+                raise Exception(f"File must be txt, but received {value}")
+        else:
+            raise Exception("It seems that {value} does not exist or is not a file.")
```
Comment (CarlosGomes98) [inline terratorch/datamodules/generic_pixel_wise_data_module.py:37]: same issue as for the scalar_label. Probably makes sense to make this a common function both use.

File: terratorch/datamodules/generic_pixel_wise_data_module.py [hunk 5]
```diff
@@ -23,6 +23,20 @@ def wrap_in_compose_is_list(transform_list):
     # set check shapes to false because of the multitemporal case
     return A.Compose(transform_list, is_check_shapes=False) if isinstance(transform_list, Iterable) else transform_list
 
+def load_from_file_or_attribute(value:list[float] | str):
```
Comment (CarlosGomes98) [inline terratorch/datamodules/generic_pixel_wise_data_module.py:26]: spacing

**Review by CarlosGomes98 — COMMENTED** [review]
the same issues were present in the `pixel_wise`. Once these are also fixed should be good to go :)

File: terratorch/datamodules/generic_pixel_wise_data_module.py (hunk 5 above)
Comment (Joao-L-S-Almeida) [reply terratorch/datamodules/generic_pixel_wise_data_module.py]: Solved.

**Review by CarlosGomes98 — APPROVED** [review]

**Pull request merged**